package.json

```diff
@@ -4,8 +4,8 @@
   "description": "A Node.js starter for the Serverless Framework with async/await and unit test support",
   "main": "handler.js",
   "scripts": {
-    "lint": "serverless-bundle eslint",
-    "test": "serverless-bundle test"
+    "lint": "babel-eslint",
+    "test": "jest ./tests/unit"
```

Comment:
are you sure you want to move away from the serverless-bundle stuff?

(not saying don't, just not sure what problem you are trying to solve)

Reply:
It's true, serverless bundle packed in a lot of awesome stuff, however, I was not able to find a good documentation on how to run lint or test, serverless-bundle at this moment only correct linting issues when you run deploy. 

As of now, I am just trying to get the ball rolling and treating each lambda as just like any other function, I didn't know we could hijack functions with spyOn and mockImplementation. Today has been a great learning experience with jest. 

Plan in the future is to definitely see if we can use serverless-bundle to lint and test.